### PR TITLE
Add nuscenes-devkit

### PR DESCRIPTION
**Pull request by wep21**
Signed-off-by: wep21 <border_goldenmarket@yahoo.co.jp>

Please add the following dependency to the rosdep database.

## Package name:

nuscenes-devkit

## Package Upstream Source:

https://github.com/nutonomy/nuscenes-devkit

## Purpose of using this:

The nuScenes and nuImages datasets are one of the most popular dataset for autonomous driving.
The devkit is useful to process the dataset.

Distro packaging links:

## Links to Distribution Packages

We can install `nuscenes devkit` via `pip`.
https://pypi.org/project/nuscenes-devkit/

- Debian: https://packages.debian.org/
  - REQUIRED
- Ubuntu: https://packages.ubuntu.com/
   - REQUIRED
- Fedora: https://src.fedoraproject.org/browse/projects/
  - IF AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
@wep21 can you please fill out the pull request template so we can review this pull request?

**Comment by wep21** [comment]
@nuclearsandwich I filled out the template. Could you review?

File: rosdep/python.yaml [hunk 1]
```diff
@@ -2978,6 +2978,22 @@ python-numpydoc:
   gentoo: [dev-python/numpydoc]
   opensuse: [python-numpydoc]
   ubuntu: [python-numpydoc]
+python-nuscenes-devkit-pip:
```
Comment (ivanpauno) [inline rosdep/python.yaml:2981]: this is a python2 entry, do you want to add this package only for python2?

if you meant to add the python3 key, it should be `python3-nuscenes-devkit-pip`

Comment (wep21) [reply rosdep/python.yaml]: @ivanpauno Fixed at https://github.com/ros/rosdistro/pull/31455/commits/dfa022734d8c3b56a12b6d9233db687fabf5b3c8.

**Review by ivanpauno — APPROVED** [review]

**Review by hidmic — APPROVED** [review]

**Pull request merged**